test/helpers

```diff
@@ -14,17 +14,50 @@ test::helpers::test_deploy() {
     #   default_modules
     #
     # Arguments:
-    #   $1 > $fixture_name: Name of the fixture to deploy.
-    #      The fixture sets the deployment conditions.
-    #      In fact, this is the name of a directory that must exist in
-    #      `BUILDPACK_DIR/test/fixtures/` and that contains the fixture code.
-    #      See `test::utils::setupFixture` for more details.
+    #   $1 > $detect_expectation
+    #      See `test::helpers::test_detect`.
     #
-    #   $2 > $detect_expectation: Expected output of `bin/compile`.
-    #      The value of this variable is compared to the actual output of the
-    #      `bin/detect` script (and should match).
+    #   $2 > $version
+    #      See `test::helpers::test_compile`.
+
+    local detect_expectation="${1}"
+    local version="${2}"
+
+    # Test that bin/detect works as expected:
+    test::helpers::test_detect "${detect_expectation}"
+
+    # Test that bin/compile works as expected:
+    test::helpers::test_compile "${version}"
+
+    # Test that all default PHP modules are available:
+    test::helpers::test_default_modules
+}
+
+test::helpers::test_detect() {
```

Comment:
suggestion: I wouldn't include "test_" in the function name as the full function name is already `test::helpers::test_detect`. Hence we know that it's a test.
```suggestion
test::helpers::detect() {
```